# request
float64 start_point_x
float64 start_point_y
float64 goal_point_x
float64 goal_point_y
---
# response
bool is_path_valid
time calculation_time